<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormatWidget</class>
 <widget class="QWidget" name="FormatWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>235</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Formatter Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Indent size:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spacingSbx">
           <property name="value">
            <number>4</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Case:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="caseCbx">
           <item>
            <property name="text">
             <string>No change</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Lower case</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Upper case</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="formatSourceCbx">
         <property name="text">
          <string>Format source sections</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>166</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
